CatalogService/Repository/CategoryRepository.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.ComponentModel.DataAnnotations;
```

Comment:
1. This repository could be async
2. I am not sure that it is a good idea to mix categories and items. We can discuss it during out 1-2-1 session